cmd/s3-gw/app.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"context"
 	"encoding/hex"
+	"github.com/nspcc-dev/neofs-s3-gw/notifications"
```

Comment:
I am not sure we need to define it on a top level. Maybe `api` package suits better? Also you didn't include this package into PR.